indico/modules/events/timetable/client/js/Timetable.module.scss

```
@@ -79,6 +81,17 @@
 
       &.selected {
         filter: opacity(0.9) brightness(0.95);
+        box-shadow: 4px 0 18px -2px $indico-blue;
+        margin-left: 4px;
+        z-index: 0;
+        border-color: $indico-blue !important;
```

we can avoid this `!important` here simply by removing the `border-color` definition in the `entryStyleGetter` whenever the entry is selected

---

```suggestion
        border-color: $indico-blue;
        border-width: 0.1em;
```
i'd also add the `border-width` to make the left-border show up, it looks weird with nothing there

---

actually, instead of all of this, why don't you make use of the `outline` attr like in the parent block?